crates/evm/evm/src/inspectors/logs.rs

```diff
@@ -67,8 +67,22 @@ fn convert_hh_log_to_event(call: HardhatConsole::HardhatConsoleCalls) -> Log {
     // Convert the parameters of the call to their string representation using `ConsoleFmt`.
     let fmt = call.fmt(Default::default());
     Log {
-        topics: vec![Console::log::SIGNATURE_HASH.to_ethers()],
+        topics: vec![Console::log::SIGNATURE_HASH],
         data: fmt.abi_encode().into(),
-        ..Default::default()
+        ..default_log()
+    }
+}
+
+fn default_log() -> Log {
+    Log {
+        address: Address::default(),
+        topics: Vec::default(),
+        data: Bytes::default(),
+        block_hash: None,
+        block_number: None,
+        transaction_hash: None,
+        transaction_index: None,
+        log_index: None,
+        removed: false,
     }
```

Comment:
this can be removed once I've merged this:

https://github.com/alloy-rs/alloy/pull/101